packages/account/src/Types/common-prop.type.ts

```diff
@@ -9,3 +9,8 @@ export type TToken = {
     scopes: string[];
     token: string;
 };
+
+export type TPoaStatusProps = {
+    needs_poi: boolean;
```

Comment:
Since needs_poi is boolean, can we make it optional? Or are we enforcing the value?

Reply:
@likhith-deriv I believe we should keep this option as required, because we need to know this value while using all necessary components